routes/person.js

```diff
@@ -14,11 +13,32 @@ module.exports = ({ elastic, config }) => ({
         mediaTypes: {
           'text/html' (request, reply) {
             const data = {
-              page: 'person',
-              title: 'Charles Babbage'
+              page: 'person'
             };
 
-            reply.view('person', Object.assign(exampleData, data));
+            elastic.get({index: 'smg', type: 'agent', id: TypeMapping.toInternal(request.params.id)}, (err, result) => {
+              if (err) {
+                if (err.status === 404) {
+                  const error = {
+                    output: {
+                      payload: {
+                        status: 404,
+                        error: 'Not found',
+                        message: 'The requested person could not be found'
+                      }
+                    }
+                  };
+
+                  return reply.view('person', {error}).code(404);
```

Comment:
No need to reply with a view here - the error handler plugin should handle the error and display an error page, simply `return reply(Boom.notFound())`